### PR TITLE
SC-6021 - fallback icons for EdgeHTML

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

Due to the fact that there are problems with webpack rendering components for icon with vue-svg-loader library, we needed to find a workaround for the legacy Edge browser which was not rendering any material icons for the new admin tables. The fallback icons will be loaded from the assets directory when no material or font awesome icon is available.
@luizzaly will provide the fallback icons that are missing at the moment.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-6021

## New Repos, NPM pakages or vendor scripts

"svg-inline-loader": "^0.8.2" is added to load svgs directly not through a component.

## Screenshots of UI changes

EdgeHTML
image

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by luizzaly — DISMISSED** [review]

**Review by mwolowyk-cap — APPROVED** [review]

**Comment by haniapalac** [comment]
@schul-cloud-bot update screenshots

File: src/components/base/BaseIcon.vue [hunk 1]
```diff
@@ -65,7 +71,7 @@ export default {
 				return icon.default;
 			} catch (error) {
 				console.error(
-					`error loading icon "${this.icon}" from "${this.source}". It might be not available.`,
+					`error loading icon "${this.icon}" from "${this.source}". It might be not available or you are using the legacy Edge browser.`,
```
Comment (luizzaly) [inline src/components/base/BaseIcon.vue:74]: can you add an a11y key here please? :)

**Review by luizzaly — CHANGES_REQUESTED** [review]
Please add an i18 key for the error message!

File: src/components/base/BaseIcon.vue (hunk 1 above)
Comment (haniapalac) [reply src/components/base/BaseIcon.vue]: done :-)

**Review by luizzaly — APPROVED** [review]

**Pull request merged**